axelor-account/src/main/java/com/axelor/apps/account/web/MoveLineController.java

```diff
@@ -57,6 +57,30 @@ public void computeAnalyticDistribution(ActionRequest request, ActionResponse re
     }
   }
 
+  public void balanceCreditDebit(ActionRequest request, ActionResponse response) {
```

Comment:
Please add a try catch to trace exceptions and refactor the the moveLine update to be made into a service.